source/wp-content/themes/wporg-main-2022/functions.php

```diff
@@ -7,6 +7,7 @@
  */
 add_action( 'wp_enqueue_scripts', __NAMESPACE__ . '\enqueue_assets' );
 add_action( 'init', __NAMESPACE__ . '\register_shortcodes' );
+add_filter( 'the_content', __NAMESPACE__ . '\prevent_arrow_emoji' );
```

Comment:
Perhaps there should be some level of priority set here, to ensure that it runs after all other filters?

```suggestion
add_filter( 'the_content', __NAMESPACE__ . '\prevent_arrow_emoji', 20 );
```